metals/src/main/scala/scala/meta/internal/metals/Embedded.scala

```diff
@@ -61,44 +60,59 @@ final class Embedded(
 }
 
 object Embedded {
-  def downloadSettings(
+  def fetchSettings(
       dependency: Dependency,
       scalaVersion: String
-  ): Settings =
-    new coursiersmall.Settings()
-      .withTtl(Some(Duration.Inf))
-      .withDependencies(List(dependency))
+  ): Fetch = {
+    val fetch = Fetch
+      .create()
+      .withDependencies(List(dependency): _*)
+      .addRepositories(
+        Repository.defaults().asScala.toList: _*
+      )
       .addRepositories(
         List(
-          coursiersmall.Repository.SonatypeReleases,
-          coursiersmall.Repository.SonatypeSnapshots
-        )
+          Repository.central(),
+          Repository.ivy2Local()
+        ): _*
       )
-      .withForceVersions(
+      .withDependencies(
         List(
-          new Dependency(
+          Dependency.of(
             "org.scala-lang",
             "scala-library",
             scalaVersion
           ),
-          new Dependency(
+          Dependency.of(
             "org.scala-lang",
             "scala-compiler",
             scalaVersion
           ),
-          new Dependency(
+          Dependency.of(
             "org.scala-lang",
             "scala-reflect",
             scalaVersion
           )
-        )
+        ): _*
       )
 
+    val envRepos = System
```

Comment:
does it fit in a single line?

Reply:
It's reworked now, so there's no envRepos.